projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-component.scss

```diff
@@ -532,6 +532,13 @@
         }
     }
 
+    @for $i from 1 through 25 {
+        @include e(tree-cell, $m: padding-level-#{$i}) {
```

Comment:
You should remove the tree-cell padding-level from the upper `for`. This is valid for all newly added `for` cycles.